src/systems/apply_link_wrench/ApplyLinkWrench.cc

```diff
@@ -182,12 +181,12 @@ void ApplyLinkWrench::Configure(const Entity &_entity,
     if (elem->HasElement("force"))
     {
       msgs::Set(msg.mutable_wrench()->mutable_force(),
-          elem->GetElement("force")->Get<math::Vector3d>());
+          elem->FindElement("force")->Get<math::Vector3d>());
```

Comment:
should be fine to use `GetElement` here since there is a `HasElement` check above to ensure that the element exists?

Reply:
We should avoid using `GetElement` since it potentially creates the element. `FindElement` should be safe to use, especially after checking with `HasElement`. `GetElement` also requires `elem` be non-const, which is not the case here.

Reply:
ah just realized that `FindElement` essentially just calls the same underlying `GetElementImp` function as `GetElement`